mirrord/cli/src/config.rs

```diff
@@ -62,7 +62,6 @@ pub(super) enum Commands {
     InternalProxy,
 
     /// Verify config file without starting mirrord.
-    #[command(hide = true)]
     VerifyConfig(VerifyConfigArgs),
```

Comment:
i am not sure if i am aware of the context here, do people really want to use it with the cli? why remove the hidden flag?

Reply:
Shouldn't be removed, missed this while I was testing some stuff.